semanticdb/semanticdb3/semanticdb3.md

```diff
@@ -130,9 +131,12 @@ SemanticDB payloads must include the version of the SemanticDB model in the
   </tr>
 </table>
 
-Code snippets referenced in `TextDocument` can be defined
-in one of three ways: 1) via a [URI](#uri) provided in `uri`,
-2) via a string provided in `text`, 3) via a combination of both.
+Code snippets referenced in `TextDocument` can be defined as any combination of
+the following three fields: 
```

Comment:
Isn't `uri` always mandatory?

Reply:
If I read it correctly, the original text implied option 2) was valid which implied only `.text` with no `.uri` is OK. I can't see it mentioned that `uri` is required. What should we do?

Reply:
Huh indeed you're right - too little sleep recently. How about not changing this paragraph at all? It's not like `md5` can be used to define code snippets - it's only verifying snippets that are defined by other means.